### PR TITLE
feat(feature-activation): add miner update doc

### PR DESCRIPTION
**Pull request by glevco**
[Rendered](https://github.com/HathorNetwork/rfcs/blob/feat/feature-activation/miner-update/projects/feature-activation/0005-miner-update-request.md)

### CODE REVIEW

File: projects/feature-activation/0005-miner-update-request.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+We of course are available for any questions or help that you might need. Thank you in advance!
+
+## Updating the Integration
+
+In order to support this new feature, a simple change must be made.
+
+### 1. Identifying where the Hathor block is generated
+
+Your current integration most certainly reads our Block template from one of our HTTP or WebSocket endpoints. The fields provided in those endpoints are the following:
```
Comment (glevco) [inline projects/feature-activation/0005-miner-update-request.md:19]: Can we be more specific here, by mentioning the specific API they use?

File: projects/feature-activation/0005-miner-update-request.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+parents_any: list[bytes]
+height: int
+score: float
+```
+
+Identify where your integration code reads this data and constructs the block.
+
+### 2. Adding `signal_bits` to the block data
+
+Currently, you integration ignores the `signal_bits` field, which didn't exist before. You likely set the first byte of the block data as `0x00`, or just use the `version` field to set the first two block bytes. You must change this so the `signal_bits` field is the first block byte, and the `version` field is the second block byte.
```
Comment (jansegre) [inline projects/feature-activation/0005-miner-update-request.md:39]: you integration -> your integration

Also worth mentioning that every previous version of the full node will ignore the value of the first byte, so using the `signal_bits` will not affect the propagation of blocks to peers that have not updated to the latest version.

**Review by jansegre — DISMISSED** [review]

File: projects/feature-activation/0005-miner-update-request.md (hunk 1 above)
Comment (glevco) [reply projects/feature-activation/0005-miner-update-request.md:19]: @jansegre ?

Comment (msbrogli) [reply projects/feature-activation/0005-miner-update-request.md]: They could also be using the ``/v1a/get_block_template` HTTP API. So we might mention it in the guide.

**Review by msbrogli — APPROVED** [review]

File: projects/feature-activation/0005-miner-update-request.md (hunk 1 above)
Comment (glevco) [reply projects/feature-activation/0005-miner-update-request.md]: Added in https://github.com/HathorNetwork/rfcs/pull/74/commits/9cb72cc16011c3ff7ccdb4ab91a2b6518dbcee52

Comment (jansegre) [reply projects/feature-activation/0005-miner-update-request.md]: I remember there being 3 different mining APIs, an older HTTP, a newer HTTP and a WebSocket, the newer HTTP and WebSocket should have the same fields (for the most part). I'll take a look to confirm this.

**Review by jansegre — APPROVED** [review]

**Pull request merged**